src/fonts/lib/Space.ttx

```diff
@@ -191,20 +149,10 @@ http://scripts.sil.org/OFL
     <cmap_format_4 platformID="0" platEncID="3" language="0">
       <map code="0x20" name="space"/><!-- SPACE -->
       <map code="0xa0" name="uni00A0"/><!-- NO-BREAK SPACE -->
-      <map code="0xeffe" name="uniEFFE"/><!-- ???? -->
-      <map code="0xefff" name="uniEFFF"/><!-- ???? -->
     </cmap_format_4>
-    <cmap_format_0 platformID="1" platEncID="0" language="0">
-      <map code="0x9" name="space"/>
-      <map code="0xd" name="space"/>
-      <map code="0x20" name="space"/>
-      <map code="0xca" name="uni00A0"/>
-    </cmap_format_0>
     <cmap_format_4 platformID="3" platEncID="1" language="0">
```

Comment:
Would it make sense to get rid of Windows specific stuff in a separate PR or do we need this?

Reply:
@kevinbarabash I haven't checked but I guess they are required in the Internet Explorer.

Reply:
[Microsoft docs](https://docs.microsoft.com/ko-kr/typography/opentype/spec/cmap) state:
> Microsoft **strongly** recommends using Unicode 'cmap' subtables for all fonts.

I'm not sure whether it means we should use `platformID="3" platEncID="1"` or not use `platformID="3"` at all.

Reply:
I guess leave it for now until we find something more conclusive.